src/Components/GoalForm/GoalForm.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { createUserGoal } from '../../actions';
+import './GoalForm.css';
+
+class GoalForm extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  handleUserInput = (event) => {
+    this.setState({ userGoal: event.target.value })
+  }
+
+  setUserGoal = (event) => {
+    this.props.createUserGoal(this.state.userGoal)
+  }
+
+  render() {
+    return(
+      <form>
+        <input
+          type='number'
```

Comment:
Let's add some text here that says something like, "Your reading goal is 25 books!"